admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+import OrderViewPageMap from './OrderViewPageMap';
+
+const $ = window.$;
+
+$(() => {
+  handlePaymentDetailsToggle();
+
+  function handlePaymentDetailsToggle() {
+    $(OrderViewPageMap.orderPaymentDetailsBtn).on('click', (event) => {
+      const $paymentDetailRow = $(event.currentTarget).closest('tr').next(':first');
+
+      if ($paymentDetailRow.hasClass('d-none')) {
```

Comment:
You can use `toggleClass` instead

Reply:
I don't like `toggleClass` because it's not predictable, it could reduce lines of code, but that wouldn't make code easier to understand imo.

Reply:
The `toggleClass` do the exact same condition as you did:
https://github.com/jquery/jquery/blob/cf84696fd1d7fe314a11492606529b5a658ee9e3/src/attributes/classes.js#L138

It's totally predictable :thinking:

Reply:
I think it's similar as with CQRS commands, `ToggleSomethingCommand` is not predictable, you don't know if it will end up one way or another. So I think having `if` is more explicit and easier to debug, but I'll follow your suggestion and use `toggleClass` as I don't think it's a big issue. :wink: